src/main.cpp

```diff
@@ -199,18 +199,18 @@ int main(const int argc, const char *const *const argv) {
                                          }) != libraryNames.end();
                  });
 
-    std::vector<std::string> extraPluginsFromEnv;
-    const auto* const extraPluginsFromEnvData = getenv("EXTRA_QT_PLUGINS");
-    if (extraPluginsFromEnvData != nullptr)
-        extraPluginsFromEnv = linuxdeploy::util::split(std::string(extraPluginsFromEnvData), ';');
+    std::vector<std::string> extraModulesFromEnv;
+    const auto* const extraModulesFromEnvData = getenv("EXTRA_QT_MODULES") ? getenv("EXTRA_QT_MODULES") : getenv("EXTRA_QT_PLUGINS");
```

Comment:
I'm not sure you want to keep old users happy, so maybe we can just rename and break usage

Reply:
We need to support both. I'm not a fan of inline ifs, so please consider splitting this up (maybe using an inline lambda you call directly, this would further allow us to print a warning that the environment variable is deprecated).

Reply:
>I'm not a fan of inline ifs
Me neither, but it was simpler an I wasn't sure what was your preference